data_diff/dbt.py

```diff
@@ -229,7 +233,7 @@ def _cloud_diff(diff_vars: DiffVars) -> None:
     if api_key is None:
         raise ValueError("API key not found, add it as an environment variable called DATAFOLD_API_KEY.")
 
-    url = "https://app.datafold.com/api/v1/datadiffs"
+    url = f"https://{host_name}/api/v1/datadiffs"
```

Comment:
@nolar do you see any reason to override the full endpoint urls? I made an assumption that the host_name is enough